protobuf/src/main/scala/magnolify/protobuf/ProtobufType.scala

```diff
@@ -117,32 +111,39 @@ object ProtobufField {
 
   def apply[T](implicit f: ProtobufField[T]): ProtobufField[T] = f
 
-  //////////////////////////////////////////////////
+  def from[T]: FromWord[T] = new FromWord[T]
```

Comment:
Same trick as in `AvroType`, etc. breaking away so only `T` needs to be supplied and `U` can be inferred from `f`.